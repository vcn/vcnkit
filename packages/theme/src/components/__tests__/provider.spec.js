import * as React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import styled from 'styled-components';

import { ThemeProvider } from '../../';

const PrimaryColorComponent = styled.div`
    background: ${props => props.theme.colors.primary};
`;

it('Renders correctly', () => {
    const tree = renderer
        .create(
            <ThemeProvider primary="#FF0000">
                <PrimaryColorComponent />
            </ThemeProvider>,
        )
        .toJSON();

    expect(tree).toMatchSnapshot();
});
