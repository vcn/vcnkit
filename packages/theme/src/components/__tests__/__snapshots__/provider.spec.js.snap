// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
.c0 {
  background: #FF0000;
}

<div
  className="c0"
/>
`;
